examples/directml/llama_v2/config_llama_v2.json

```diff
@@ -895,9 +895,6 @@
                     "/norm/Mul"
                 ]
             }
-        },
```

Comment:
also, this example still needs a merging pass. This model is a composite model so it follows this flow `https://github.com/microsoft/Olive/blob/main/olive/passes/olive_pass.py#L357`. There will still be composite onnx model here.

Reply:
Pulled out the change to OptimumMerge & OptimumModelHandler. PR focus on introducing merge decoders for OnnxConversion only.